package.json

```diff
@@ -45,7 +45,7 @@
     "@dcl/content-hash-tree": "^1.1.4",
     "@dcl/hashing": "^3.0.1",
     "@dcl/schemas": "^11.4.0",
-    "@dcl/urn-resolver": "^3.3.1",
+    "@dcl/urn-resolver": "https://sdk-team-cdn.decentraland.org/@dcl/urn-resolver/branch/feat/add-polygon-amoy-support/dcl-urn-resolver-3.3.2-8681282457.commit-f52dd9e.tgz",
```

Comment:
We should release URN-Resolver and upgrade its version to latest, right?

Reply:
yep, I'll upgrade the version to the latest as soon as the `urn-resolver` is released